pkgs/development/python-modules/loguru/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, isPy2, pytest, colorama }:
+
+buildPythonPackage rec {
+  pname = "loguru";
+  version = "0.3.0";
+disabled = isPy2
```

forgot a semicolon :(, my bad.
```suggestion
  disabled = isPy2;
```

---

Sorry, I should have looked more carefully.